### DISCUSSION TITLE
Wyze Bulb Color (new product)

### DISCUSSION THREAD

**Discussion by sean898** [SecKatie/ha-wyzeapi#114] [General]
The color bulb was just announced yesterday. What are the chances these will be supported by this repo/HA? The price is very appealing and I'm thinking about pre-ordering.

**Comment by kdoh82** [comment]
If the bulbs work just like the non color bulbs then it should work with
this integration i would think.

**Comment by SecKatie** [comment]
I have preordered the bulbs and plan on supporting them. Please see my latest post on BuyMeACoffee for more info! https://www.buymeacoffee.com/joshmulliken/wyze-bulb-color
